fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckContextualOnboarding.kt

```diff
@@ -116,6 +122,16 @@ fun ReviewQualityCheckContextualOnboarding(
                         onPrivacyPolicyClick()
                     },
                 ),
+            ),
+            style = FirefoxTheme.typography.body2,
+            linkTextDecoration = TextDecoration.Underline,
+        )
+
+        Spacer(modifier = Modifier.height(24.dp))
```

Comment:
❓ Guessing this is to increase the accidental tap to the other link?

Reply:
That was my impression, yes. I got this spacing from the updated figma